lib/widgets/message_list.dart

```diff
@@ -892,6 +893,16 @@ class MessageWithPossibleSender extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final message = item.message;
+    bool messageEdited = false, messageMoved = false;
+    if(message.editHistory != null) {
+      if((message.editHistory?[message.editHistory!.length-1].topic !=
+          message.editHistory?[message.editHistory!.length-1].prevTopic) ||
+          (message.editHistory?[message.editHistory!.length-1].stream !=
+          message.editHistory?[message.editHistory!.length-1].prevStream)){
+            messageMoved = true;}
+      if(message.editHistory?[message.editHistory!.length-1].prevContent!=null){
+        messageEdited = true;
```

Comment:
What does this do if the message is edited, and then it's moved? What's the desired behavior in that case?

Reply:
If a message has been edited and moved, or has been moved and then edited - I believe it should prioritize showing edited over moved in both cases. Please correct me if I'm wrong.

Reply:
Yeah, that's right.